Makefile

```diff
@@ -20,7 +20,9 @@ release:
 .PHONY: release-static
 release-static:
 	RUSTFLAGS="-C target-feature=+crt-static" cargo build --release --target x86_64-unknown-linux-gnu
-	ldd target/x86_64-unknown-linux-gnu/release/conmonrs | grep -q "statically linked"
+	strip -s target/x86_64-unknown-linux-gnu/release/conmonrs
+	ldd target/x86_64-unknown-linux-gnu/release/conmonrs# | grep -q "not a dynamic executable"
```

Comment:
It may output on stderr

Reply:
`grep -qE '(statically linked)|(not a dynamic executable)'` would allow us to check for both